src/quo/components/inputs/input/view.cljs

```diff
@@ -150,7 +150,7 @@
               :auto-complete       :password
               :secure-text-entry   (not @password-shown?)
               :right-icon          {:style-fn  style/password-icon
-                                    :icon-name (if @password-shown? :i/hide :i/reveal)
+                                    :icon-name (if @password-shown? :i/hide-password :i/reveal)
```

Comment:
just wondering if you double checked this with design? seems a bit weird for them to update the icon in one place (I more so mean the reference key to the icon). Did they update other uses of this across the designs?

Reply:
What do you mean in one place? On the icon resources page, there is a comment on the icon that they update icon's name (`hide` to `hide-password`).

Reply:
perfect, sounds good 👍